.travis.yml

```diff
@@ -12,7 +12,8 @@ before_script:
 
 script:
     - nc -z localhost 8080
-    - docker-compose -f docker-compose.yaml run ci
+    - docker-compose -f docker-compose.yaml run ci vendor/bin/phpunit
+    - docker-compose -f docker-compose.yaml run ci vendor/bin/phpcs
```

Comment:
Would be interesting to try https://docs.travis-ci.com/user/build-stages/share-docker-image/.

Reply:
Would they run in parallel?

Reply:
Jobs in each stage would be. I am not sure if this is all on the same node or not (as the latter would require `docker-compose up` to be moved together with the `nc` smoke test)